packages/algob/src/internal/deployer.ts

```diff
@@ -284,6 +306,101 @@ class DeployerBasicMode {
   async ensureCompiled (name: string, force?: boolean, scTmplParams?: SCParams): Promise<ASCCache> {
     return await this.algoOp.ensureCompiled(name, force, scTmplParams);
   }
+
+  /**
+   * Check if ASA info exist for asset id / string,
+   * First: search for ASAInfo in checkpoints
+   * Case 1: If it exist check if that info is deleted or not by checking deleted boolean
+   * If deleted boolean is true throw error
+   * else, pass
+   * Case 2: If it doesn't exist, pass
+   * @param asset asset index or asset name
+   */
+  private assertIfASAExist (asset: string | number): void {
```

Comment:
let's remove the `If` infix. In all assertion libraries we have `assert<noun><verb>`

Reply:
ok